keyboards/stoutgat/rules.mk

```diff
@@ -0,0 +1,26 @@
+# MCU name
+MCU = atmega32a
+
+# Processor frequency
+F_CPU = 16000000
+
+# Bootloader selection
+BOOTLOADER = USBasp
+
+# Build Options
+#   change yes to no to disable
+#
+BOOTMAGIC_ENABLE = lite     # Virtual DIP switch configuration
+MOUSEKEY_ENABLE = yes       # Mouse keys
+EXTRAKEY_ENABLE = yes       # Audio control and System control
+CONSOLE_ENABLE = no         # Console for debug
+COMMAND_ENABLE = no         # Commands for debug and configuration
+# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
+# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
+NKRO_ENABLE = no            # USB Nkey Rollover
+BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
+RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
+BLUETOOTH_ENABLE = no       # Enable Bluetooth
+AUDIO_ENABLE = no           # Audio output
+ENCODER_ENABLE  = yes
```

Comment:
```suggestion
ENCODER_ENABLE  = yes

LAYOUTS = 65_iso
```

Reply:
This is not actually a 65% ISO layout, there are two extra keys for the encoder switches.

Reply:
I have more suggestions, but I had to jump into a meeting.

Reply:
I'm still new to the whole layout bit, how would the community layouts get applied?

Reply:
@vattern https://docs.qmk.fm/#/feature_layouts

This page should cover most of how to use the community layout feature, and how ti implement it.

The big thing is that the layout needs to match what is in /layouts/default/ or /layouts/community/ for the specific layout in question.

Reply:
I've added 65_iso and 65_ansi